src/config.rs

```diff
@@ -363,8 +363,8 @@ pub enum Config {
     /// MsgId of webxdc map integration.
     WebxdcIntegration,
 
-    /// Iroh secret key.
-    IrohSecretKey,
+    /// Enable webxdc realtime features.
+    EnableRealtime,
```

Comment:
This should be `WebxdcRealtimeEnabled`:
- Other config keys have "Enabled" at the end (`E2eeEnabled` etc.).
- Public functions are called `*_webxdc_realtime_*()`, so we should have "Webxdc" here to make it clear they are related.